scripts/cmake/ParseCMakeArgs.cmake

```diff
@@ -0,0 +1,19 @@
+# Captures not-yet cached CMake variables.
+# On first CMake run via cmake-cli this works as expected.
+# Once the variables are cached this will not work anymore.
```

Comment:
... and the use-case for this is overriding certain settings...?

Reply:
No I just want to have the info on the CMake configuration inside the OGS exe. As I said this only works of you specify everything on the first CMake run (as we in Jenkins, so the Jenkins provided binaries contain the correct information).